<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #192231;
color: #D3D3D3;
border-bottom: 2px solid #4C5C75;
border-radius: 30px;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="menu_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0A1628;
color: #FFFFFF;
border-bottom: 3px solid #00FF7F;
border-radius: 30px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/logo football.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string>Competition</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_4">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <fontweight>Thin</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(158, 158, 158);</string>
    </property>
    <property name="text">
     <string>Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/cab2d94a7ebc1fe3660958e20762166c.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(40, 40, 40);
border: 2px solid #4C5C75;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4C5C75;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4C5C75;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>971</width>
      <height>621</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
    border: 2px solid #2c2f3f;
    background: #1e1f29;
    border-radius: 5px;
}

QTabBar::tab {
    background: #1e1f29;
    color: #ffffff;
    padding: 8px 16px;
    border: 1px solid #2c2f3f;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:selected {
    background: #2c2f3f;
    color: #ffffff;
    font-weight: bold;
    border-bottom: 2px solid #0078d7;
}

QTabBar::tab:hover {
    background: #323546;
    color: #ffffff;
}

QTabBar::tab:!selected {
    margin-top: 2px;
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add Match</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>761</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0F111A;
color: #FFFFFF;
font-family: 'Segoe UI', Arial, sans-serif;
font-size: 12px;
border-radius: 15px;
padding: 15px;
spacing: 2px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0">
        <widget class="QLineEdit" name="Stadium">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="team2">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QDateTimeEdit" name="Date">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Stadium</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="team1">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Team_2</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QPushButton" name="Confirm">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(50, 150, 250, 0.8);
color: white;
font-size: 12px;
border-radius: 5px;
padding: 6px 12px;
margin-left:500px;
margin-right:20px;
margin-bottom:10px;</string>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;
background-color:transparent;
</string>
         </property>
         <property name="text">
          <string>Referee's FullName</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="Referee_name">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Competiton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color: #A0A8B7;
font-weight: bold;
padding-bottom: 5px;background-color:transparent;</string>
         </property>
         <property name="text">
          <string>Team_1</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QComboBox" name="Comp">
         <property name="styleSheet">
          <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="auto_match">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>-140</x>
        <y>514</y>
        <width>743</width>
        <height>38</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(50, 150, 250, 0.8);
color: white;
font-size: 12px;
border-radius: 5px;
padding: 6px 12px;
margin-left:500px;
margin-right:20px;
margin-bottom:10px;</string>
      </property>
      <property name="text">
       <string>Auto Match's Schedule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Display Match</string>
     </attribute>
     <widget class="QTableView" name="allmatches">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>741</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style for the QTableWidget */
QTableView#allmatches {
    background-color: #1E2633; /* Match tab pane and form widget */
    border: 2px solid #2A3A4A; /* Subtle blue-gray border */
    gridline-color: #3A4A5A; /* Light grid lines */
    alternate-background-color: #252A38; /* Alternating row color */
    color: #E0E6ED; /* Light gray text */
    selection-background-color: #00FF7F; /* Green selection */
    selection-color: #1A2633; /* Dark blue text on selection */
    border-radius: 5px;
}
QTableView#allmatches::item {
    padding: 5px;
    border: none; /* Remove individual cell borders */
}
QTableView#allmatches::item:focus {
    outline: none; /* Remove focus outline */
}

/* Style for the table header */
QHeaderView::section {
    background-color: #252A38; /* Match alternating row color */
    color: #E0E6ED;
    border: 1px solid #2A3A4A;
    padding: 4px;
    font-weight: bold;
    border-radius: 0;
}
QTableView#allmatches QTableCornerButton::section {
    background-color: #252A38;
    border: 1px solid #2A3A4A;
}</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    background-color: #1E1E2D;  /* Dark matte background */


 

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="search">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">    background-color: #141821;  /* Slightly lighter for contrast */
    border: 2px solid #cccccc;  /* Keep borders subtle */
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 14px;
    color: white;</string>
       </property>
      </widget>
      <widget class="QLabel" name="sortem">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/trier.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>15</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>11</width>
          <height>11</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/search.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Delete">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
border-radius: 5px ;
color: white;</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modify">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #192231;
border-radius: 5px ;
color: white;</string>
      </property>
      <property name="text">
       <string>Modify</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #192231;
border-radius: 5px ;
color: white;</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #192231;
border-radius: 5px ;
color: white;</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QTableView" name="Matches_sum">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>921</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style for the QTableWidget */
QTableView#allmatches {
    background-color: #1E2633; /* Match tab pane and form widget */
    border: 2px solid #2A3A4A; /* Subtle blue-gray border */
    gridline-color: #3A4A5A; /* Light grid lines */
    alternate-background-color: #252A38; /* Alternating row color */
    color: #E0E6ED; /* Light gray text */
    selection-background-color: #00FF7F; /* Green selection */
    selection-color: #1A2633; /* Dark blue text on selection */
    border-radius: 5px;
}
QTableView#allmatches::item {
    padding: 5px;
    border: none; /* Remove individual cell borders */
}
QTableView#allmatches::item:focus {
    outline: none; /* Remove focus outline */
}

/* Style for the table header */
QHeaderView::section {
    background-color: #252A38; /* Match alternating row color */
    color: #E0E6ED;
    border: 1px solid #2A3A4A;
    padding: 4px;
    font-weight: bold;
    border-radius: 0;
}
QTableView#allmatches QTableCornerButton::section {
    background-color: #252A38;
    border: 1px solid #2A3A4A;
}</string>
      </property>
     </widget>
     <widget class="QTableView" name="refrees">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>921</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style for the QTableWidget */
QTableView#allmatches {
    background-color: #1E2633; /* Match tab pane and form widget */
    border: 2px solid #2A3A4A; /* Subtle blue-gray border */
    gridline-color: #3A4A5A; /* Light grid lines */
    alternate-background-color: #252A38; /* Alternating row color */
    color: #E0E6ED; /* Light gray text */
    selection-background-color: #00FF7F; /* Green selection */
    selection-color: #1A2633; /* Dark blue text on selection */
    border-radius: 5px;
}
QTableView#allmatches::item {
    padding: 5px;
    border: none; /* Remove individual cell borders */
}
QTableView#allmatches::item:focus {
    outline: none; /* Remove focus outline */
}

/* Style for the table header */
QHeaderView::section {
    background-color: #252A38; /* Match alternating row color */
    color: #E0E6ED;
    border: 1px solid #2A3A4A;
    padding: 4px;
    font-weight: bold;
    border-radius: 0;
}
QTableView#allmatches QTableCornerButton::section {
    background-color: #252A38;
    border: 1px solid #2A3A4A;
}</string>
      </property>
     </widget>
     <widget class="QWidget" name="weather_impact" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>921</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#customWidget {
    background-color: #1E2633;
    border: 2px solid #2A3A4A;
    color: #E0E6ED;
    border-radius: 5px;
}
QWidget#customWidget QLabel {
    color: #E0E6ED;
    background-color: transparent;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chart">
     <attribute name="title">
      <string>Chart</string>
     </attribute>
     <widget class="QWidget" name="goals_chart" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>851</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Style for the QTableWidget */
QTableView#allmatches {
    background-color: #1E2633; /* Match tab pane and form widget */
    border: 2px solid #2A3A4A; /* Subtle blue-gray border */
    gridline-color: #3A4A5A; /* Light grid lines */
    alternate-background-color: #252A38; /* Alternating row color */
    color: #E0E6ED; /* Light gray text */
    selection-background-color: #00FF7F; /* Green selection */
    selection-color: #1A2633; /* Dark blue text on selection */
    border-radius: 5px;
}
QTableView#allmatches::item {
    padding: 5px;
    border: none; /* Remove individual cell borders */
}
QTableView#allmatches::item:focus {
    outline: none; /* Remove focus outline */
}

/* Style for the table header */
QHeaderView::section {
    background-color: #252A38; /* Match alternating row color */
    color: #E0E6ED;
    border: 1px solid #2A3A4A;
    padding: 4px;
    font-weight: bold;
    border-radius: 0;
}
QTableView#allmatches QTableCornerButton::section {
    background-color: #252A38;
    border: 1px solid #2A3A4A;
}</string>
      </property>
     </widget>
     <widget class="QComboBox" name="chartOptionComboBox">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>20</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
      </property>
      <item>
       <property name="text">
        <string>Goals per Competition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Goals per Team</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="line_up_widget">
     <attribute name="title">
      <string>Line up</string>
     </attribute>
     <widget class="QComboBox" name="Line_up">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1C1F2A;
border: 1px solid #2A2D3A;
border-radius: 5px;
padding: 5px;
color: white;
margin-left: 15px;
margin-right:15px;</string>
      </property>
      <item>
       <property name="text">
        <string>4-3-3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4-4-2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>621</width>
        <height>551</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/staduim.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
